controllers/defaults/utils.go

```diff
@@ -24,14 +38,31 @@ func GetProfileDaemonResources(name string, sc *ocsv1.StorageCluster) corev1.Res
 	}
 	resourceProfile := sc.Spec.ResourceProfile
 	resourceProfile = strings.ToLower(resourceProfile)
+	var resourceRequirements corev1.ResourceRequirements
 	switch resourceProfile {
 	case "lean":
-		return LeanDaemonResources[name]
+		resourceRequirements = LeanDaemonResources[name]
 	case "balanced":
-		return BalancedDaemonResources[name]
+		resourceRequirements = BalancedDaemonResources[name]
 	case "performance":
-		return PerformanceDaemonResources[name]
+		resourceRequirements = PerformanceDaemonResources[name]
 	default:
-		return BalancedDaemonResources[name]
+		resourceRequirements = BalancedDaemonResources[name]
+	}
+	if runtime.GOARCH == "s390x" {
```

Comment:
Could you define a const for `"s390x"` since it's used in multiple places?

Reply:
Done